tests/unit/utils/get-header-test.js

```diff
@@ -0,0 +1,44 @@
+/* jshint expr:true */
```

Comment:
Leftover from the `ember generate` process. Wish it were a simple thing to `extend` blueprints, but alas, [the issue is open](https://github.com/ember-cli/blueprint/issues/2).